### PR TITLE
doc: add more details to GSG for the hybrid scenario

### PR DESCRIPTION
**Pull request by gvancuts**
Add a number of steps and details that were not called
out in the "Getting Started Guide for ACRN Hybrid Mode". Those
are not obvious to the first-time or novice user so the user
guide was hard to follow and confusing. At a high-level:
* How to build Zephyr
* How to install ACRN
* How to install the ACRN kernel

Tracked-On: #5992
Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

### CODE REVIEW

File: doc/tutorials/using_hybrid_mode_on_nuc.rst [hunk 1]
```diff
@@ -22,15 +22,63 @@ as shown in :numref:`hybrid_scenario_on_nuc`.
...
 .. rst-class:: numbered-step
 
+Prepare the Zephyr image
+************************
+
+Prepare the Zephyr kernel that you will run in VM0 later.
+
+- Follow step 1 from the :Ref:`using_zephyr_as_uos` instructions
+
+  .. note:: We only need the binary Zephyr kernel, not the entire ``zephyr.img``
```
Comment (dbkinder) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst:44]: Step 1 in the referenced doc creates a ``zephyr.elf`` file.  How do we get the ``zephyr.bin`` file referenced in this next step?

**Review by dbkinder — CHANGES_REQUESTED** [review]
These instructions have the developer building on the target board and not on their host computer.  We should at least mention that here since it could be confusing that developer would need all the build tools installed on the target board.

File: doc/tutorials/using_hybrid_mode_on_nuc.rst (hunk 1 above)
Comment (gvancuts) [reply doc/tutorials/using_hybrid_mode_on_nuc.rst:44]: It's built at the same time and put in the same folder.

Comment (dbkinder) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst:44]: In that case, let's change this to:

```suggestion
  .. note:: We only need the binary Zephyr kernel, not the entire ``zephyr.img``.  This step builds a ``zephyr.elf`` and the ``zephyr.bin`` image we'll use in the next step.
```

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Comment by wenlingz** [comment]
@gvancuts make doc fail, please check:

New errors/warnings found, please fix them:
==============================================

/home/acrn/workspace/make_doc/pme-yocto-build-make-doc-02/acrn-hypervisor/build/doc/rst/tutorials/using_hybrid_mode_on_nuc.rst:15: WARNING: image file not readable: tutorials/images/ACRN-Hybrid.png.png

Makefile:65: recipe for target 'html' failed
make[1]: *** [html] Error 1
make[1]: Leaving directory '/home/acrn/workspace/make_doc/pme-yocto-build-make-doc-02/acrn-hypervisor/doc'
Makefile:129: recipe for target 'doc' failed
make: *** [doc] Error 2
+ error make doc
++ timestamp
++ date +%T
+ echo '16:40:56 ERROR: make doc'
16:40:56 ERROR: make doc
+ exit 1

**Comment by gvancuts** [comment]
> @gvancuts make doc fail, please check:

Oops... sorry, fixed!

File: doc/tutorials/using_hybrid_mode_on_nuc.rst [hunk 2]
```diff
@@ -18,19 +18,72 @@ The following guidelines
 describe how to set up the ACRN hypervisor hybrid scenario on the Intel NUC,
 as shown in :numref:`hybrid_scenario_on_nuc`.
 
+.. note::
+
+   All build operations are done directly on the target. Building the artefacts (ACRN hypervisor, kernel, tools and Zephyr)
```
Comment (dbkinder) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst:23]:
```suggestion
   All build operations are done directly on the target. Building the artifacts (ACRN hypervisor, kernel, tools and Zephyr)
```

File: doc/tutorials/using_hybrid_mode_on_nuc.rst [hunk 3]
```diff
@@ -18,19 +18,72 @@ The following guidelines
...
   Intel NUC.
 
 .. rst-class:: numbered-step
 
+Prepare the Zephyr image
+************************
+
+Prepare the Zephyr kernel that you will run in VM0 later.
+
+- Follow step 1 from the :Ref:`using_zephyr_as_uos` instructions
```
Comment (dbkinder) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst:47]:
```suggestion
- Follow step 1 from the :ref:`using_zephyr_as_uos` instructions
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
couple more tweaks...

File: doc/tutorials/using_hybrid_mode_on_nuc.rst (hunk 2 above)
Comment (gvancuts) [reply doc/tutorials/using_hybrid_mode_on_nuc.rst:23]: Good catch ;-))

**Review by dbkinder — APPROVED** [review]
LGTM now, thanks.
